src/ape/managers/config.py

```diff
@@ -19,6 +33,65 @@ class DeploymentConfig(ConfigItem):
     contract_type: str
 
 
+class DependencyConfig(ConfigItem):
+    name: str
+    _compilers: "CompilerManager" = None  # type: ignore
+
+    @abstractmethod
+    def extract_manifest(self) -> PackageManifest:
+        """
+        Download and extract the package manifest for this config entry.
+
+        Returns:
+            PackageManifest
+        """
+
+
+class IPFSDependency(DependencyConfig):
+    ipfs: str
+
+    def extract_manifest(self) -> PackageManifest:
+        response = requests.get(self.ipfs)
```

Comment:
this probably won't work lol... maybe we skip this one for now, and do `GitLab` instead (since it's basically the same)

Reply:
Good call! Let me find an example make the switch

Reply:
was picturing this would be `ipfs: Qm....` so just the content id. Can probably use a public resolver for now

Reply:
Actually, infura has one too